src/components/firebase.js

```diff
@@ -0,0 +1,19 @@
+import firebase from "firebase/app"
+import "firebase/firestore"
+
+var firebaseConfig = {
+  apiKey: "AIzaSyAECA70-2ckL4XnoLIFFeN_6JDOuE8zSG0",
```

Comment:
deze ApiKey wil je geheim houden, nu zit ie in je client code dus kan iedereen hem zien en misbruiken. Ik zou voorstellen om hem in een .env file te zetten die niet gecommit wordt

Reply:
Dit heb ik gedaan